AJOAuth2/ViewControllers/LeftViewController.m

```diff
@@ -60,6 +62,12 @@ - (void)viewWillAppear:(BOOL)animated {
         
         // Email Address
         _emailLabel.text = user.emailAddress;
+        
+        // ImageView
+        if (user.avatarImageURLString.length > 0)
+            _avatarImageView.image = [UIImage imageWithData:[Helper avatarImageUrl:user.avatarImageURLString]];
```

Comment:
oh yeah, definitely needed! Will take care in another PR. Hopefully _SDWebImage_ integration for caching stuff.